compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -194,6 +198,8 @@
     private BLangAnonymousModelHelper anonymousModelHelper;
     private SemanticAnalyzer semanticAnalyzer;
     private boolean nonErrorLoggingCheck = false;
+    private Set<String> modifierFunctions = new HashSet<>();
```

Comment:
This could be a static variable

Reply:
fixed by d9d1222ebd2f15e8eb8cc80333d36825d92b488a